erpnext/hr/doctype/hr_settings/hr_settings.py

```diff
@@ -24,12 +24,12 @@ def validate_password_policy(self):
 		if self.email_salary_slip_to_employee and self.encrypt_salary_slips_in_emails:
 			if not self.password_policy:
 				frappe.throw(_("Password policy for Salary Slips is not set"))
-	
+
 	def on_update(self):
 		self.toggle_rounded_total()
 		frappe.clear_cache()
 
 	def toggle_rounded_total(self):
-		self.disable_rounded_total = cint(self.disable_rounded_total)
+		self.disable_rounded_total = cint(self.get("disable_rounded_total"))
```

Comment:
This is not the right fix. Ideally, we should write a patch to set disable_rounded_total based on global_defaults. And in that patch the field will be reloaded. We will add the patch.

By the way, which patch is breaking because of this?

Reply:
This is breaking for us on a fork, but the patch exists for review on a PR up at #19709.

More specifically, this file: https://github.com/frappe/erpnext/pull/19709/files#diff-4c931075e81c4c445c73fcf15dd55af1.